roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -164,7 +164,7 @@ helm_enabled: false
 istio_enabled: false
 registry_enabled: false
 enable_network_policy: false
-local_volumes_enabled: false
+local_volume_provisioner_enabled: false
```

Comment:
I'm wondering for retro-compat if we should do: 
```
# deprecated will be removed
local_volumes_enabled: false        
local_volume_provisioner_enabled: {{ local_volumes_enabled }}
```
?

Reply:
I guess you're right. Fixed in #e69979d5